src/vs/workbench/contrib/workspace/browser/workspace.contribution.ts

```diff
@@ -890,3 +908,19 @@ class WorkspaceTrustTelemetryContribution extends Disposable implements IWorkben
 
 Registry.as<IWorkbenchContributionsRegistry>(WorkbenchExtensions.Workbench)
 	.registerWorkbenchContribution(WorkspaceTrustTelemetryContribution, LifecyclePhase.Restored);
+
+function logWorkspaceTrustDialogResult(telemetryService: ITelemetryService, eventName: string, duration: number, result: IShowResult): void {
```

Comment:
unfortunately the gdpr tooling cannot parse this because it doesn't know how to find the event name

Reply:
I see. I have pushed a commit to address this.